src/app/AttributeCache.cpp

```diff
@@ -219,5 +219,59 @@ CHIP_ERROR AttributeCache::GetStatus(const ConcreteAttributePath & path, StatusI
     return CHIP_NO_ERROR;
 }
 
+uint32_t AttributeCache::OnUpdateDataVersionFilterList(DataVersionFilterIBs::Builder & aDataVersionFilterIBsBuilder,
+                                                       DataVersionFilter * apDataVersionFilterList,
+                                                       size_t aDataVersionFilterListSize)
```

Comment:
Please pass a Span, not a pointer/length pair.

Reply:
Done, thanks